src/api/controllers/groupController.ts

```diff
@@ -0,0 +1,53 @@
+import { BaseController } from './baseController.js';
+import { RequestWithGroup } from '../../types/requests.js';
+import { NextFunction, Request, Response } from 'express';
+import { GroupService } from '../../services/groupService.js';
+import { BaseGroup } from '../../types/group.js';
+import { EntityNotFoundError } from '../../core/errors/entityNotFoundError.js';
+
+
+export class GroupController extends BaseController {
+    private groupService: GroupService;
+
+    constructor(groupServiceInjected: GroupService) {
```

Comment:
that could be done like `constructor(private groupServiceInjected: GroupService)` and removing like #10 and #14.